include/seqan/graph_msa/graph_align_tcoffee_library.h

```diff
@@ -707,12 +731,26 @@ appendSegmentMatches(StringSet<TString, Dependent<TSpec> > const& str,
                      TScores& scores,
                      LocalPairwiseLibrary)
 {
+    appendSegmentMatches(str, pList, score_type, matches, scores, LocalPairwiseLibrary(), OrdinaryAlignment());
+}
+
+//////////////////////////////////////////////////////////////////////////////
+
+template<typename TString, typename TSpec, typename TSize2, typename TSpec2, typename TScore, typename TSegmentMatches, typename TScoreValues>
+inline void
+appendSegmentMatches(StringSet<TString, Dependent<TSpec> > const& str,
+                     String<TSize2, TSpec2> const& pList,
+                     TScore const& score_type,
+                     TSegmentMatches& matches,
+                     TScoreValues& scores,
+                     LocalPairwiseLibrary,
+                     OrdinaryAlignment)
+{
```

Comment:
If ```matches``` is the output parameter it should come first in the argument list of the function. Always put output arguments at the beginning. The same for ```scores```, if it is changed. Otherwise make it ```const &```. Please always pass tags as
```cpp 
Tag const & /*descr*/
```
. In this way if the parameter type is generic the const is kept out of the type. In this case it really won't matter, since it is not a generic type, but just for consistency pass it as ```const &``` while you write the name of the variable in comments as depicted above.